packages/pages/src/common/src/module/types.ts

```diff
@@ -1,10 +1,21 @@
+import React from "react";
+
 /**
  * The type definition for the module's default function.
  *
  * @public
  */
 export type Module = () => React.JSX.Element;
 
+/**
+ * The type to include in any module file.
+ */
+export interface ModuleModule {
```

Comment:
Do we need this type at all? It's not something someone would use since it defines the whole file which you can't actually use in any practical sense. We could just remove it to avoid this weird name.

Reply:
We use this in the call to SSRLoadModule which populates it with everything from the .tsx file. It's only used in one place so we might be able to just use <any> and parse instead.

Reply:
Ok maybe rename it to something like `ModuleDefinition`